sources/packages/api/src-sql/Types/Create-note-types.sql

```diff
@@ -0,0 +1 @@
+CREATE TYPE sims.note_types AS ENUM ('General', 'Program', 'Restriction', 'System');
```

Comment:
what about `All Notes`? and you can give `System Actions` instead of `System`
![image](https://user-images.githubusercontent.com/77353155/145623806-6ad71491-161c-419d-9bff-60175223f20b.png)

Reply:
System modified to System Actions.